queries.sql

```diff
@@ -6,3 +6,29 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Vet clinic database: query and update animals table
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;

```

Comment:
- [ ] Great work on your query to update the `species` column in the animal table! 🐾👏 However, I'd like to highlight a specific requirement from the project guidelines: it explicitly states that after updating the `species` column to `unspecified`, verify that the change was made successfully. Then roll back the change and `verify` that the `species` columns went back to the state before the transaction. 🔄

  - Upon reviewing your query, I noticed that you successfully updated the `species` column to `unspecified` within a transaction. However, it appears that you missed the verification steps outlined in the project guidelines. These verification steps are crucial to ensuring the integrity of the data and the correctness of your SQL query.

    **To meet the project requirements, please consider adding the following verification steps to your query:**

     1. After the update, verify that the change was made successfully. You can do this by executing a SELECT statement to confirm that the `species` column now contains the value `unspecified`. 🧐

     2. After rolling back the transaction, verify that the `species` columns have reverted to their state before the transaction. Again, you can use a SELECT statement to check that the `species` column values are back to their original values. 🔄

Adding these verification steps will not only ensure that your query meets the project requirements but also provide a robust and reliable way to confirm the correctness of your data manipulation.

Happy coding! 😊🚀🍀